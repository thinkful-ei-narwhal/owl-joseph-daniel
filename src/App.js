import React from 'react';
import ParticipantList from './participant-list.js';
import Stage from './stage';
import ChatLog from './chat-log';
import STORE from './store';
import './App.css';

export default function App() {

  const p = <ParticipantList participants={STORE.participants}/>;
  const s = <Stage participants={STORE.participants}/>;
  const c = <ChatLog chatEvents ={STORE.chatEvents}/>;
  
  return (
    <main className="App">
      <section>{p}</section>
      <section>{s}</section>
      <section>{c}</section>
  </main>
  );
};
