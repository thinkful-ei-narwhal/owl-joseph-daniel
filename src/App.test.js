import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from './App';
import STORE from './store';



describe('<App />', () => {
  it('smoke test', () => {
    const div = document.createElement('div')
    ReactDOM.render(<App />, div);
    // if something is wrong, itd fail here
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders this UI as expected', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});