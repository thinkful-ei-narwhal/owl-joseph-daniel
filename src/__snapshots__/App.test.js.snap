// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders this UI as expected 1`] = `
<main
  className="App"
>
  <section>
    <div
      className="participant-list"
    >
      <div
        className="Participant"
      >
        <img
          alt="Koren Templeton"
          src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
        />
        <h3>
          Koren Templeton
        </h3>
        <p>
          in session
        </p>
      </div>
      <div
        className="Participant"
      >
        <img
          alt="Caty Flucker"
          src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
        />
        <h3>
          Caty Flucker
        </h3>
        <p>
          in session
        </p>
      </div>
      
      <div
        className="Participant"
      >
        <img
          alt="Frank Runciman"
          src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
        />
        <h3>
          Frank Runciman
        </h3>
        <p>
          in session
        </p>
      </div>
      <div
        className="Participant"
      >
        <img
          alt="Ashla Attwool"
          src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
        />
        <h3>
          Ashla Attwool
        </h3>
        <p>
          in session
        </p>
      </div>
    </div>
  </section>
  <section>
    <div
      className="participant-list"
    >
      <div
        className="StagePart"
      >
        <h3>
          Koren Templeton
        </h3>
        <img
          alt="Koren Templeton"
          src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
        />
      </div>
      <div
        className="StagePart"
      >
        <h3>
          Ashla Attwool
        </h3>
        <img
          alt="Ashla Attwool"
          src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
        />
      </div>
    </div>
  </section>
  <section>
    <div
      className="chat-list"
    >
      <hr />
      <div
        className="Chat-event"
      >
        <img
          alt="Axe Kubicka"
          src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
        />
        <h3>
          Axe Kubicka
        </h3>
        <p>
          7:50
        </p>
        <p>
          Hello world
        </p>
        <p>
          
        </p>
      </div>
      <div
        className="Chat-event"
      >
        <img
          alt="Caty Flucker"
          src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
        />
        <h3>
          Caty Flucker
        </h3>
        <p>
          
        </p>
        <p>
          
        </p>
        <p>
          gave a thumbs up
        </p>
      </div>
      <div
        className="Chat-event"
      >
        <img
          alt="Frank Runciman"
          src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
        />
        <h3>
          Frank Runciman
        </h3>
        <p>
          
        </p>
        <p>
          
        </p>
        <p>
          gave a thumbs down
        </p>
      </div>
      <div
        className="Chat-event"
      >
        <img
          alt="Koren Templeton"
          src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
        />
        <h3>
          Koren Templeton
        </h3>
        <p>
          
        </p>
        <p>
          
        </p>
        <p>
          raised their hand
        </p>
      </div>
      <div
        className="Chat-event"
      >
        <img
          alt="Koren Templeton"
          src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
        />
        <h3>
          Koren Templeton
        </h3>
        <p>
          
        </p>
        <p>
          
        </p>
        <p>
          clapped
        </p>
      </div>
      <div
        className="Chat-event"
      >
        <img
          alt="Koren Templeton"
          src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
        />
        <h3>
          Koren Templeton
        </h3>
        <p>
          
        </p>
        <p>
          
        </p>
        <p>
          joined the room
        </p>
      </div>
      <div
        className="Chat-event"
      >
        <img
          alt="Ashla Attwool"
          src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
        />
        <h3>
          Ashla Attwool
        </h3>
        <p>
          
        </p>
        <p>
          
        </p>
        <p>
          left the room
        </p>
      </div>
      <div
        className="Chat-event"
      >
        <img
          alt="Axe Kubicka"
          src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
        />
        <h3>
          Axe Kubicka
        </h3>
        <p>
          
        </p>
        <p>
          
        </p>
        <p>
          joined the stage
        </p>
      </div>
      <div
        className="Chat-event"
      >
        <img
          alt="Axe Kubicka"
          src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
        />
        <h3>
          Axe Kubicka
        </h3>
        <p>
          
        </p>
        <p>
          
        </p>
        <p>
          left the stage
        </p>
      </div>
    </div>
  </section>
</main>
`;
